docs/methoddocs/types.md

```diff
@@ -10,14 +10,43 @@
 ## Signatures
 
 ```{eval-rst}
-.. automodule:: ape.types.signatures
+.. autoclass:: ape.types.signatures.SignableMessage
     :members:
     :show-inheritance:
+    :exclude-members: model_fields
+```
+
+```{eval-rst}
+.. autoclass:: ape.types.signatures.MessageSignature
+    :members:
+    :show-inheritance:
+    :exclude-members: model_fields
+```
+
+```{eval-rst}
+.. autoclass:: ape.types.signatures.TransactionSignature
+    :members:
+    :show-inheritance:
+    :exclude-members: model_fields
```

Comment:
code simplify this part by merging https://github.com/ApeWorX/ape/pull/1830 first